src/components/Metadata/MetadataCard.tsx

```diff
@@ -52,196 +20,92 @@ export const MetadataCard: React.FC<MetadataCardProps> = ({
   onChange,
 }) => {
   const intl = useIntl();
-  const [expanded, setExpanded] = React.useState(false);
-  const classes = useStyles();
+  const title = isPrivate
+    ? {
+        id: "ETHnjq",
+        defaultMessage: "Private Metadata",
+        description: "header",
+      }
+    : {
+        id: "VcI+Zh",
+        defaultMessage: "Metadata",
+        description: "header",
+      };
```

Comment:
are these ids generated or manually set?

Reply:
Generated and were there already, I only moved them above to better readability
https://github.com/saleor/saleor-dashboard/pull/4125/files#diff-a61b9e9b0a890fe1d70cf3f9279d9569be625eeafbe833ef9e28bc637398880dL72-L82